src/components/text-note/footer/EmojiPickerPanel.tsx

```
@@ -1,23 +1,37 @@
+  }, [onEmojiSelect])
+
   return (
-    <EmojiPicker
-      className="h-87.5"
-      onEmojiSelect={({ emoji }) => {
-        onEmojiSelect(emoji)
+    <emoji-picker
+      ref={emojiPickerRef}
+      class="[--background:transparent] [--border-size:0]"
+      onClick={(event) => {
+        event.stopPropagation()
```

ESLint

[error] Unsafe member access .stopPropagation on an `any` value.

(ts/no-unsafe-member-access)